### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by finny-dev** [armastudiosuk/issue-tracker#5]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

If you do certain things in a short period of time in the radial menu the NUI breaks. - And you cannot open the menu again unless you relog

### Expected behaviour

It to always open i guess?

### Steps to reproduce

It usually happens when u click something in the MET Police section and then spam the X in the middle to close and it will glitch and you have to press ESC for the menu to close and it wont open again.

### ARMA User ID

101

**Comment by kwt** [comment]
If you can simply press escape, I doubt this is an actual issue and just occurs with you attempting to make use of multiple UIs at once

**Comment by kwt** [comment]
Not responded for a week, going to close this for now. I've offered solutions, please open a new issue with an accompanying video if it happens frequently.
